### PR TITLE
fix for direction capture

### PR DESCRIPTION
**Pull request by likewei92**
arc-left/right wasn't working in frontend2; this change makes it work

### CODE REVIEW

File: src/arc.directive.ts [hunk 1]
```diff
@@ -57,22 +64,22 @@ export class ArcDirective implements OnInit, OnDestroy, IArcDirective {
...
 
   @Input('arc-up')
-  public set arcUp(target: Element) {
+  public set arcUp(target: any) {
     this.handlers.push(createDirectionCapture(Direction.UP, target));
   }
 
   @Input('arc-down')
-  public set arcDown(target: Element) {
+  public set arcDown(target: any) {
```
Comment (connor4312) [inline src/arc.directive.ts]: Please avoid using `any` types! Maybe you want `Element | HTMLElement | ElementRef`?

**Comment by likewei92** [comment]
@connor4312 got rid of any

**Comment by connor4312** [comment]
What's the use case where an element is nested like that?

**Comment by likewei92** [comment]
In FE2, when you do something like 
`<bui-tab-label arc #channelsAll/>
<bui-tab-label arc [arc-left]="channelsAll"/>`
the argument passed into arc-left actually has the element nested as shown. This was causing arc overrides to not work in FE2

**Comment by connor4312** [comment]
Okay, what's probably happening is that you're passing an instance of the TabLabelComponent into the directive. That's won't really work; there's an `element` in the component class in this case but it's not a general solution. In this case you should probably pass `channelsAll.element` and make sure that the element is publicly accessible in the component.

**Comment by likewei92** [comment]
ah I see. oops didn't realize you can do that

**Pull request closed without merging**